src/vocabularyChecker/detectWrongSyntax.ts

```diff
@@ -0,0 +1,79 @@
+import json from "./AllowedVocab.json"
+
+/**
+ * Checks, if all words used, are allowed.
+ * 
+ * 
+ * 
+ */
+
+function preprocessingText(text: string){
+    var cleantext = eliminatePunctuation(text);
+    var textarray = transformTextToStringArray(cleantext);
+    return textarray;
+}
+
+export function checkInput(text: string){
+    let preprocessedText : string[] = preprocessingText(text);
+    let issue : string[] = []; //collect all wrong words in a stringarray
+    for (const word of preprocessedText) {
+        if (!json.foo.includes(word)) {//word not included in AllowedVocab.json
+            console.log(word + " ist kein erlaubtes Wort. Bitte verwenden Sie ein anderes!"); //for debugging purposes. The user should get a message on screen telling him what's wrong.
+            issue.push(word);
+        }
+    }
+    let allWrongWords :string = "";
+    allWrongWords += issue.join //Join all wrong words to one String
```

Comment:
This works?!? I assume `issue.join` concatenates without any delimiters - not sure if that's what you want? Generally, even though this might work, you should always call methods in javascript, with parameters. So don't use `issue.join`, but `issue.join(", ")` instead.

Usually, I'd avoid string concatenation using `+=` - it's much nicer to do this instead:
```
const errorMessage = `${invalidWords.join(", ")} sind nicht erlaubte Wörter. Bitte verwenden Sie andere!`;
```
For bonus points, you should only create an error message when there *are* invalid words - so turn it into 
```
const errorMessage = problematicWords.length > 0 ? `${invalidWords.join(", ")} sind nicht...` : undefined;
if (errorMessage !== undefined) {
  console.log(errorMessage);
}
```
and ideally create a separate function `logInvalidWords(words: string[])` for it.